webapp/certification/views.py

```diff
@@ -15,6 +15,89 @@
 )
 
 
+def certification_hardware_details(canonical_id, release):
+
+    try:
+        models = api.certified_models(canonical_id=canonical_id)["objects"][0]
+    except KeyError:
+        abort(404)
+
+    model_releases = api.certified_model_details(
+        canonical_id=canonical_id, limit="0"
+    )["objects"]
+
+    # Release section
+    release_details = next(
+        (
+            detail
+            for detail in model_releases
+            if detail["certified_release"] == release
+        ),
+        None,
+    )
+    if not release_details:
+        abort(404)
+
+    model_devices = api.certified_model_devices(
+        canonical_id=canonical_id, limit="0"
+    )["objects"]
+
+    hardware_details = {}
+    for device in model_devices:
+        device_info = {
+            "name": (
+                f"{device['make']} {device['name']}"
+                f" {device['subproduct_name']}"
+            ),
+            "bus": device["bus"],
+            "identifier": device["identifier"],
+        }
+
+        category = device["category"]
+        if category not in ["BIOS", "USB"]:
+            category = category.capitalize()
+
+        if category not in hardware_details:
+            hardware_details[category] = []
+
+        hardware_details[category].append(device_info)
+
+    # Components section
+    components = {}
+    for category in ["video", "processor", "network", "wireless"]:
```

Comment:
If you look in the suggestion I made the categories were capitalised, no need to do it on the frontend.

Reply:
Sorry, just talked to Beth about this, we don't need a components section anymore. The `hardware_details` is enough